Q3-2024 community board blog post (#2191)
writethedocs/www: docs/blog/2024-Q3-community-board.rst

.. post:: Aug 15, 2024
   :tags: community board, news, report
   :author: Mikey Ariel

Write the Docs Community Board Quarterly Update Q3-2024
=======================================================

> Greetings, documentarians! Mikey Ariel here, chairperson of the WTD community board. I'm happy to share with you the third quarterly update for 2024. 

Reviewer: ```suggestion
Greetings, documentarians! Mikey Ariel here, chairperson of the WTD community board. I'm happy to share the third quarterly update for 2024 with you. 
```

To catch up on previous quarterly updates, check out the `community board blog posts <https://www.writethedocs.org/blog/archive/tag/community-board/>`_ archive. 

Q3 quarterly meeting highlights
-------------------------------

Our Q3 meeting covered several initiatives that the team leads have been working on, as well as some conference-related topics: 

> * Following the Trello pricing change, to ensure that specialized teams have a stable platform to manage their tasks, we decided to create Trello workspaces for each conference as well as the meetup team. Each conference chair can now manage a standalone workspace with multiple boards, which also gives them more flexibility to add and remove team members for each year. The GitHub projects pilot was cancelled, and the meetups team will move back to Trello under their own dedicated workspace.

Author: ```suggestion
* **Team resources:** Following the Trello pricing change, to ensure that specialized teams have a stable platform to manage their tasks, we decided to create Trello workspaces for each conference as well as the meetup team. Each conference chair can now manage a standalone workspace with multiple boards, which also gives them more flexibility to add and remove team members for each year. The GitHub projects pilot was cancelled, and the meetups team will move back to Trello under their own dedicated workspace.
```

> * Hopin usage is officially deprecated, and the Atlantic 2024 virtual conference will be held on Venueless as the new chosen platform. Venueless is connected to the Pretalx platform where we already manage our calls for proposals, and their support team has been responsive and helpful during the pilot stage. Ticketholders for the Atlantic 2024 conference will receive instructions on how to log into the conference using their Ti.to ticket contact information.  

Reviewer: ```suggestion
* **Conference infrastructure:** Hopin usage is officially deprecated, and the Atlantic 2024 virtual conference will be held on Venueless. Venueless is connected to the Pretalx platform where we already manage our calls for proposals, and their support team has been responsive and helpful during the pilot stage. Ticketholders for the Atlantic 2024 conference will receive instructions on how to log into the conference using their Ti.to ticket contact information.  
```

> * **Conference infrastructure:** Hopin usage is officially deprecated, and the Atlantic 2024 virtual conference will be held on Venueless. Venueless is connected to the Pretalx platform where we already manage our calls for proposals, and their support team has been responsive and helpful during the pilot stage. Ticketholders for the Atlantic 2024 conference will receive instructions on how to log into the conference using their Ti.to ticket contact information.  

Author: ```suggestion
* **Atlantic infrastructure:** Hopin usage is officially deprecated, and the Atlantic 2024 virtual conference will be held on Venueless. Venueless is connected to the Pretalx platform where we already manage our calls for proposals, and their support team has been responsive and helpful during the pilot stage. Ticketholders for the Atlantic 2024 conference will receive instructions on how to log into the conference using their Ti.to ticket contact information.  
```

> * The the meetups team has been pursuing an audit of the active meetup.com accounts that WTD has been supporting. This is a part of the ongoing effort to ensure that active meetups get the support they need while ensuring that the WTD organization is not paying for meetup groups that have been inactive for an extended period of time. Rose has been compiling an updated set of guidelines for WTD sponsorship of meetup.com accounts, and will update the team guidance when it is finalized. 

Author: ```suggestion
* **Meetups:** The meetups team has been pursuing an audit of the active meetup.com accounts that WTD has been supporting. This is a part of the ongoing effort to ensure that active meetups get the support they need while ensuring that the WTD organization is not paying for meetup groups that have been inactive for an extended period of time. Rose has been compiling an updated set of guidelines for WTD sponsorship of meetup.com accounts, and will update the team guidance when it is finalized. 
```

> * **Meetups:** The meetups team has been pursuing an audit of the active meetup.com accounts that WTD has been supporting. This is a part of the ongoing effort to ensure that active meetups get the support they need while ensuring that the WTD organization is not paying for meetup groups that have been inactive for an extended period of time. Rose has been compiling an updated set of guidelines for WTD sponsorship of meetup.com accounts, and will update the team guidance when it is finalized. 

Author: ```suggestion
* **Meetups:** The meetups team has been pursuing an audit of the active meetup.com accounts that WTD has been supporting. This is a part of the ongoing effort to ensure that active meetups get the support they need while ensuring that the WTD organization is not paying for meetup groups that have been inactive for an extended period of time. Rose Williams has been compiling an updated set of guidelines for WTD sponsorship of meetup.com accounts, and will update the team guidance when it is finalized. 
```

> * The Newsletter team happily added a new author to its roster, and Aaron is working with Eric Holscher to investigate additional sponsorship options due to high demand from organizations. They are also in contact with some of our community members who offered to help with refactoring some of the topic indexing of the WTD website and documentation guides. 

Author: ```suggestion
* **Newsletter:** The Newsletter team happily added a new author to its roster, and Aaron is working with Eric Holscher to investigate additional sponsorship options due to high demand from organizations. They are also in contact with some of our community members who offered to help with refactoring some of the topic indexing of the WTD website and documentation guides. 
```

> * **Newsletter:** The Newsletter team happily added a new author to its roster, and Aaron is working with Eric Holscher to investigate additional sponsorship options due to high demand from organizations. They are also in contact with some of our community members who offered to help with refactoring some of the topic indexing of the WTD website and documentation guides. 

Author: ```suggestion
* **Newsletter:** The Newsletter team happily added a new author to its roster, and Aaron Collier is working with Eric Holscher to investigate additional sponsorship options due to high demand from organizations. They are also in contact with some of our community members who offered to help with refactoring some of the topic indexing of the WTD website and documentation guides. 
```

> * The 2023 salary survey results have been `published on our website <https://www.writethedocs.org/surveys/salary-survey/2023/>`_, and the responses have been more robust than ever. Mikey has been reducing her involvement in non-board projects, so the primary board contact for the survey will be Sam Wright. Stay tuned for the 2024 survey, which will be launched in early September and will remain open until the end of the year. 

Reviewer: I feel like a few of these bullet points are non sequitors, but it could just me be. 

For example, new newsletter member, and sponsorship. Maybe it would be more clear to label the paragraphs as Meetups, Newsletter, etc?

Author: I was thinking that each bullet point reflects either a project or a team update. We usually don't have as many points, so prefixing them would probably help with skimming

> * The 2023 salary survey results have been `published on our website <https://www.writethedocs.org/surveys/salary-survey/2023/>`_, and the responses have been more robust than ever. Mikey has been reducing her involvement in non-board projects, so the primary board contact for the survey will be Sam Wright. Stay tuned for the 2024 survey, which will be launched in early September and will remain open until the end of the year. 

Author: ```suggestion
* **Salary survey:** The 2023 salary survey results have been `published on our website <https://www.writethedocs.org/surveys/salary-survey/2023/>`_, and the responses have been more robust than ever. Mikey has been reducing her involvement in non-board projects, so the primary board contact for the survey will be Sam Wright. Stay tuned for the 2024 survey, which will be launched in early September and will remain open until the end of the year. 
```

> * The WTD Slack is continuing to enjoy a stable and relatively low volume of spammer blog posts, and the moderators have not had to handle as many reports as this time last year. Janine will continue to coordinate with her team on any new issues that might come up.

Reviewer: Some names written in full "Sam Wright" and some in short "Janine"

> * The WTD Slack is continuing to enjoy a stable and relatively low volume of spammer blog posts, and the moderators have not had to handle as many reports as this time last year. Janine will continue to coordinate with her team on any new issues that might come up.

Author: ```suggestion
* **Slack moderation:** The WTD Slack is continuing to enjoy a stable and relatively low volume of spammer blog posts, and the moderators have not had to handle as many reports as this time last year. Janine will continue to coordinate with her team on any new issues that might come up.
```

> * **Slack moderation:** The WTD Slack is continuing to enjoy a stable and relatively low volume of spammer blog posts, and the moderators have not had to handle as many reports as this time last year. Janine will continue to coordinate with her team on any new issues that might come up.

Author: ```suggestion
* **Slack moderation:** The WTD Slack is continuing to enjoy a stable and relatively low volume of spammer blog posts, and the moderators have not had to handle as many reports as this time last year. Janine Chan will continue to coordinate with her team on any new issues that might come up.
```

> * The Portland conference team has already started working on the 2025 edition, and dates have been published on our website. To check out the conference dates, see `Write the Docs Portland 2025 <https://www.writethedocs.org/conf/portland/2025/>`_.

Author: ```suggestion
* **Portland:** The Portland conference team has already started working on the 2025 edition, and dates have been published on our website. To check out the conference dates, see `Write the Docs Portland 2025 <https://www.writethedocs.org/conf/portland/2025/>`_.
```

> * **Portland:** The Portland conference team has already started working on the 2025 edition, and dates have been published on our website. To check out the conference dates, see `Write the Docs Portland 2025 <https://www.writethedocs.org/conf/portland/2025/>`_.

Reviewer: ```suggestion
* **Portland:** The Portland conference team has already started working on `Write the Docs Portland 2025 <https://www.writethedocs.org/conf/portland/2025/>`_, and dates have been published on our website.
```

> * The Australia conference team is happy to celebrate its 8th birthday! The Call for Proposals is closing this week, but tickets are available and all thee information is available on the `Write the Docs Australia 2024 <https://www.writethedocs.org/conf/australia/2024/>`_ website.

Reviewer: ```suggestion
* The Australia conference team is happy to celebrate its 8th birthday! The Call for Proposals is closing this week, but tickets are available and all the information is available on the `Write the Docs Australia 2024 <https://www.writethedocs.org/conf/australia/2024/>`_ website.
```

> * The Australia conference team is happy to celebrate its 8th birthday! The Call for Proposals is closing this week, but tickets are available and all the information is available on the `Write the Docs Australia 2024 <https://www.writethedocs.org/conf/australia/2024/>`_ website.

Author: ```suggestion
* **Australia:** The Australia conference team is happy to celebrate its 8th birthday! The Call for Proposals is closing this week, but tickets are available and all the information is available on the `Write the Docs Australia 2024 <https://www.writethedocs.org/conf/australia/2024/>`_ website.
```

> * **Australia:** The Australia conference team is happy to celebrate its 8th birthday! The Call for Proposals is closing this week, but tickets are available and all the information is available on the `Write the Docs Australia 2024 <https://www.writethedocs.org/conf/australia/2024/>`_ website.

Reviewer: ```suggestion
* **Australia:** The Australia conference team is celebrating its 8th birthday! The Call for Proposals is closing this week, but tickets are available and all the information is available on the `Write the Docs Australia 2024 <https://www.writethedocs.org/conf/australia/2024/>`_ website.
```
